src/Microsoft.Dnx.Runtime/NuGet/SemanticVersion.cs

```diff
@@ -207,7 +160,7 @@ private static bool TryParseInternal(string version, bool strict, out SemanticVe
                 }
             }
 
-            semVer = new SemanticVersion(NormalizeVersionValue(versionValue), specialVersion, version.Replace(" ", ""));
```

Comment:
version.Replace(" ", "") was originally passed in to be stored as OriginalString, which I think is redundant? I removed the constructor that required the OriginalString and removed this argument here. version is not used in the function after this point so it should be safe.